src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -3540,8 +3540,7 @@ public void TestTargetTypedDefault()
             var expr = this.ParseExpression(text, TestOptions.ExperimentalParseOptions);
```

Comment:
I changed it to use 7.1 explicitly (instead of Experimental, which now means Latest). But I didn't move it because this parsing test relies on base type (`ParsingTests`) for some infrastructure.

I already had a test for diagnostics in 7.0 scenario:
```C#
   [Fact]
        public void TestCSharp7()
        {
            string source = @"
class C
{
    static void Main()
    {
        int x = default;
    }
}
";
            var comp = CreateCompilationWithMscorlib(source);
            comp.VerifyDiagnostics(
                // (6,17): error CS8107: Feature 'default literal' is not available in C# 7. Please use language version 7.1 or greater.
                //         int x = default;
                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion7, "default").WithArguments("default literal", "7.1").WithLocation(6, 17)
                );
        }
```

Let me know if that addresses your concern.